src/app/[locale]/(default)/events/page.tsx

```diff
@@ -21,5 +25,65 @@ export default async function EventsPage({
 }) {
   const { locale } = await params;
   setRequestLocale(locale);
-  return <div>This should be events page</div>;
+  const t = await getTranslations('events');
+
+  return (
+    <>
+      <h1 className='my-4'>{t('title')}</h1>
+      <h2 className='my-2'>{t('activeEvents')}</h2>
+      {events.slice(0, 1).map((event) => (
+        <EventCard
+          key={event.id}
+          id={event.id}
+          title={event.title}
+          subheader={event.subheader}
+          description={event.description}
+          imagePath={event.imagePath}
+          startTime={new Date(event.startTime)}
+          endTime={new Date(event.endTime)}
+          internal={event.internal}
+          locale={locale}
+          _active
+        />
+      ))}
+      <h2 className='my-4'>{t('upcomingEvents')}</h2>
+      <div className='grid grid-cols-1 gap-2 lg:grid-cols-2'>
+        {events.slice(1, 5).map((event) => (
+          <EventCard
+            key={event.id}
+            id={event.id}
+            title={event.title}
+            subheader={event.subheader}
+            description={event.description}
+            imagePath={event.imagePath}
+            startTime={new Date(event.startTime)}
+            endTime={new Date(event.endTime)}
+            internal={event.internal}
+            locale={locale}
+            wrapperClassName='lg:last:odd:col-span-2'
+            cardClassName='h-full'
+          />
+        ))}
+      </div>
+      <h2 className='my-4'>{t('pastEvents')}</h2>
+      <div className='grid grid-cols-1 gap-2 lg:grid-cols-2'>
+        {events.slice(5).map((event) => (
+          <EventCard
```

Comment:
Instead of passing in so many props it would be much nicer to pass in just the event pbject